docs/submission/validating-molecular-data.md

```diff
@@ -0,0 +1,447 @@
+
+This validation client will check: 
+- Data submitted in the metadata payload aligns with the BAM/FASTQ files.
+- Each submission is for a single sample.
+- Each payload is formatted according to the [data validation rules](/docs/submission/molecular-data-prep#file-and-data-validation-rules) stated above.
+
+
+### Installing the Validation Client 
+
+<!---  Tabs start here -->
+
+<Tabs
+  groupId="operating-systems"
+  defaultValue="client"
+  values={[
+    { label: 'Client', value: 'client', },
+    { label: 'Docker', value: 'docker', },
+  ]
+}>
+<TabItem value="client">
+
+A required pre-requisite is to have the [`samtools`](http://www.htslib.org/) package installed, which is used to retrieve BAM header information. To read JSON/JSONL outputs in a friendly manner, we also suggest installing the `jq` package. 
+
+```bash
+sudo apt install samtools
+sudo apt install jq
+```
+
+Once samtools is installed, clone the repository:
+
+```bash
+git clone https://github.com/icgc-argo/seq-tools.git
+```
+Using pip, install `seq-tools`: 
+
+```bash
+cd seq-tools
+pip3 install -r requirements.txt  # installs Python dependencies
+pip3 install . # installs seq-tools
+```
+
+You can verify that the installation worked by checking the version: 
+```bash
+seq-tools -v
+```
+To update the client to the latest version, pull the latest version of the repository and run: 
+```bash
+cd seq-tools
+pip3 install .
+```
+
+</TabItem>
+<TabItem value="docker">
+
+```bash
+docker pull quay.io/icgc-argo/seq-tools:1.0.0
+alias seq-tools-in-docker="docker run -t -v `pwd`:`pwd` -w `pwd` quay.io/icgc-argo/seq-tools:1.0.0 seq-tools"
+```
+
+</TabItem>
+</Tabs>
+
+<!---  Tabs end here -->
+
+
+### Structured Directory Validation
+One method to quickly complete validation is to structure your data in a pre-determined format that `seq-tools` is expecting. Under a submission directory, compile each submission as a folder, with the JSON payload and the associated data files in the same folder. 
+
+Each submission folder **must** contain:
+- The metadata payload as a `.json` file. Only 1 `.json` file must be present in the submission folder.
+- All molecular data files associated to that payload, or symlinks to their location.
+
+For example: 
+![Example Directory Structure](/assets/submission/molecular_submission_validation_directory.png)
+
+Once your data is ready, use `seq-tools` from the submission directory to validate all of the data: 
+
+<!---  Tabs start here -->
+
+<Tabs
+  groupId="operating-systems"
+  defaultValue="client"
+  values={[
+    { label: 'Client', value: 'client', },
+    { label: 'Docker', value: 'docker', },
+  ]
+}>
+<TabItem value="client">
+
+```bash
+seq-tools validate submission-directory/*/*.json  
+```
+
+</TabItem>
+<TabItem value="docker">
+
+```bash
+cd ..
+seq-tools-in-docker validate -d submission-directory/*/*.json 
```

Comment:
Can we just use the folder name in the above example, like:
seq-tools-in-docker validate -d molecular-data-submission/*/*.json